run-devel.sh

```diff
@@ -50,6 +50,8 @@ fi
 ROS_DISTRO=$ROS_DISTRO \
   VOLUMES_FOLDER=$VOLUMES_FOLDER \
   CONTAINER_NAME=$CONTAINER_NAME \
+  SSH_AUTH_SOCK_HOST_PATH="$SSH_AUTH_SOCK" \
+  SSH_AUTH_SOCK_CONTAINER_PATH="/home/orise/.ssh-agent/ssh-agent.sock" \
```

Comment:
shouldn't we guarantee that `ssh-agent` is using this specific socket?

Reply:
This is the socket inside the container

Reply:
The socket in the host is defined by `$SSH_AUTH_SOCK_HOST_PATH="$SSH_AUTH_SOCK`